examples/src/main/java/tachyon/examples/BasicNonByteBufferOperations.java

```diff
@@ -72,16 +71,13 @@ public Boolean call() throws Exception {
     tachyonConf.set(Constants.MASTER_HOSTNAME, mMasterLocation.getHost());
     tachyonConf.set(Constants.MASTER_RPC_PORT, Integer.toString(mMasterLocation.getPort()));
     ClientContext.reset(tachyonConf);
-    TachyonFileSystem tachyonClient = TachyonFileSystem.TachyonFileSystemFactory.get();
+    FileSystem tachyonClient = FileSystem.Factory.get();
     write(tachyonClient);
     return read(tachyonClient);
   }
 
-  private void write(TachyonFileSystem tachyonClient) throws IOException, TachyonException {
-    OutStreamOptions clientOptions =
-        new OutStreamOptions.Builder(ClientContext.getConf()).setWriteType(mWriteType).build();
-    FileOutStream fileOutStream =
-        getOrCreate(tachyonClient, mFilePath, mDeleteIfExists, clientOptions);
+  private void write(FileSystem tachyonClient) throws IOException, TachyonException {
```

Comment:
Will rename after all the logical changes are done. As a side note, do you prefer `fileSystem` or `fs`?